### ISSUE TITLE
Wrong path for assets

### ISSUE THREAD

**Issue by karakanb** [apache/airflow#19647]
### Apache Airflow version

2.2.2 (latest released)

### Operating System

macOS

### Deployment

Docker-Compose

### Deployment details

Running things locally with pyenv as well as Breeze.

### What happened

I have pulled the latest version last night but that doesn't seem to fix the issue, neither by local pyenv nor by Docker. 

I cannot copy separate logs from tmux, iTerm doesn't seem to allow that, but I can show the logs from the boot process of the breeze env:
```
Good version of docker 20.10.8.
f321cbfd33eeb8db8effc8f9bc4c3d5317758927da26abeb4c08b14fad09ff6b
f321cbfd33eeb8db8effc8f9bc4c3d5317758927da26abeb4c08b14fad09ff6b

No need to pull the image. Yours and remote cache hashes are the same!


The CI image for Python python:3.8-slim-buster image likely needs to be rebuild
The files were modified since last build: setup.py setup.cfg Dockerfile.ci .dockerignore scripts/docker/compile_www_assets.sh scripts/docker/common.sh scripts/docker/install_additional_dependencies.sh scripts/docker/install_airflow.sh scripts/docker/install_airflow_dependencies_from_branch_tip.sh scripts/docker/install_from_docker_context_files.sh scripts/docker/install_mysql.sh airflow/www/package.json airflow/www/yarn.lock airflow/www/webpack.config.js airflow/ui/package.json airflow/ui/yarn.lock

WARNING!!!!:Make sure that you rebased to latest upstream before rebuilding or the rebuild might take a lot of time!


Please confirm pull and rebuild image CI-python3.8 (or wait 4 seconds to skip it). Are you sure? [y/N/q]
The answer is 'no'. Skipping pull and rebuild image CI-python3.8.




                                  @&&&&&&@
                                 @&&&&&&&&&&&@
                                &&&&&&&&&&&&&&&&
                                        &&&&&&&&&&
                                            &&&&&&&
                                             &&&&&&&
                           @@@@@@@@@@@@@@@@   &&&&&&
                          @&&&&&&&&&&&&&&&&&&&&&&&&&&
                         &&&&&&&&&&&&&&&&&&&&&&&&&&&&
                                         &&&&&&&&&&&&
                                             &&&&&&&&&
                                           &&&&&&&&&&&&
                                      @@&&&&&&&&&&&&&&&@
                   @&&&&&&&&&&&&&&&&&&&&&&&&&&&&  &&&&&&
                  &&&&&&&&&&&&&&&&&&&&&&&&&&&&    &&&&&&
                 &&&&&&&&&&&&&&&&&&&&&&&&         &&&&&&
                                                 &&&&&&
                                               &&&&&&&
                                            @&&&&&&&&
            @&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&
           &&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&
          &&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&



     @&&&@       &&  @&&&&&&&&&&&   &&&&&&&&&&&&  &&            &&&&&&&&&&  &&&     &&&     &&&
    &&& &&&      &&  @&&       &&&  &&            &&          &&&       &&&@ &&&   &&&&&   &&&
   &&&   &&&     &&  @&&&&&&&&&&&&  &&&&&&&&&&&   &&          &&         &&&  &&& &&& &&@ &&&
  &&&&&&&&&&&    &&  @&&&&&&&&&     &&            &&          &&@        &&&   &&@&&   &&@&&
 &&&       &&&   &&  @&&     &&&@   &&            &&&&&&&&&&&  &&&&&&&&&&&&     &&&&   &&&&

&&&&&&&&&&&&   &&&&&&&&&&&&   &&&&&&&&&&&@  &&&&&&&&&&&&   &&&&&&&&&&&   &&&&&&&&&&&
&&&       &&&  &&        &&&  &&            &&&                  &&&&    &&
&&&&&&&&&&&&@  &&&&&&&&&&&&   &&&&&&&&&&&   &&&&&&&&&&&       &&&&       &&&&&&&&&&
&&&        &&  &&   &&&&      &&            &&&             &&&&         &&
&&&&&&&&&&&&&  &&     &&&&@   &&&&&&&&&&&@  &&&&&&&&&&&&  @&&&&&&&&&&&   &&&&&&&&&&&


                               Use CI image.

                               Branch name:            main
                               Docker image:           ghcr.io/apache/airflow/main/ci/python3.8:latest
                               Airflow source version: 2.3.0.dev0
                               Python version:         3.8
                               Backend:                mysql 5.7

####################################################################################################
                                  Airflow Breeze CHEATSHEET

####################################################################################################
 Adding breeze to your path:
    When you exit the environment, you can add sources of Airflow to the path - you can
    run breeze or the scripts above from any directory by calling 'breeze' commands directly

     export PATH=${PATH}:"/Users/burakkarakan/Code/anything-else/airflow"

####################################################################################################
 Port forwarding:

   Ports are forwarded to the running docker containers for webserver and database
     * 12322 -> forwarded to Airflow ssh server -> airflow:22
     * 28080 -> forwarded to Airflow webserver -> airflow:8080
     * 25555 -> forwarded to Flower dashboard -> airflow:5555
     * 25433 -> forwarded to Postgres database -> postgres:5432
     * 23306 -> forwarded to MySQL database  -> mysql:3306
     * 21433 -> forwarded to MSSQL database  -> mssql:1443
     * 26379 -> forwarded to Redis broker -> redis:6379

   Here are links to those services that you can use on host:
     * ssh connection for remote debugging: ssh -p 12322 airflow@127.0.0.1 pw: airflow
     * Webserver: http://127.0.0.1:28080
     * Flower:    http://127.0.0.1:25555
     * Postgres:  jdbc:postgresql://127.0.0.1:25433/airflow?user=postgres&password=airflow
     * Mysql:     jdbc:mysql://127.0.0.1:23306/airflow?user=root
     * Redis:     redis://127.0.0.1:26379/0
####################################################################################################
  You can setup autocomplete by running 'breeze setup-autocomplete'


####################################################################################################
  You can toggle ascii/cheatsheet by running:
      * breeze toggle-suppress-cheatsheet
      * breeze toggle-suppress-asciiart

####################################################################################################




Checking resources.

* Memory available 5.9G. OK.
* CPUs available 3. OK.
WARNING!!!: Not enough Disk space available for Docker.
At least 40 GBs recommended. You have 28G

WARNING!!!: You have not enough resources to run Airflow (see above)!
Please follow the instructions to increase amount of resources available:
   Please check https://github.com/apache/airflow/blob/main/BREEZE.rst#resources-required for details


Good version of docker-compose: 1.29.2

WARNING: The ENABLE_TEST_COVERAGE variable is not set. Defaulting to a blank string.
Creating network "docker-compose_default" with the default driver
Creating volume "docker-compose_sqlite-db-volume" with default driver
Creating volume "docker-compose_postgres-db-volume" with default driver
Creating volume "docker-compose_mysql-db-volume" with default driver
Creating volume "docker-compose_mssql-db-volume" with default driver
Creating docker-compose_mysql_1 ... done
Creating docker-compose_airflow_run ... done

Airflow home: /root/airflow
Airflow sources: /opt/airflow
Airflow core SQL connection: mysql://root@mysql/airflow?charset=utf8mb4


Using already installed airflow version


No need for www assets recompilation.

===============================================================================================
             Checking integrations and backends
===============================================================================================
MySQL: OK.
-----------------------------------------------------------------------------------------------
-----------------------------------------------------------------------------------------------

Starting Airflow

Your dags for webserver and scheduler are read from /files/dags directory
which is mounted from your <AIRFLOW_SOURCES>/files/dags folder


You can add /files/airflow-breeze-config directory and place variables.env
In it to make breeze source the variables automatically for you


You can add /files/airflow-breeze-config directory and place .tmux.conf
in it to make breeze use your local .tmux.conf for tmux
```

The logs say `No need for www assets recompilation.` which signals that the assets are already up-to-date. However, when I visit the page, the files are not there:
<img width="687" alt="image" src="https://user-images.githubusercontent.com/16530606/142189803-92dc5e9f-c940-4272-98a1-e5845344b62d.png">

- When I search the whole directory with `find . -name 'ab_filters.js'`, there's no file, which means the issue is not with the browser cache.

- Just in case there was another race condition, I ran the following command again to see if it'd generate the file for some reason: `./breeze initialize-local-virtualenv --python 3.8` but the result of the `find . -name 'ab_filters.js' ` is still empty. 

- Then I ran `./airflow/www/compile_assets.sh` but that didn't make a difference as well, `find` is still empty.
Here's the output from the `compile_assets.sh`:
```
❯ ./airflow/www/compile_assets.sh
yarn install v1.22.17
[1/4] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.56s.
yarn run v1.22.17
$ NODE_ENV=production webpack --colors --progress
23% building 26/39 modules 13 active ...www/node_modules/babel-loader/lib/index.js??ref--5!/Users/burakkarakan/Code/anything-else/airflow/airflow/www/static/js/datetime_utils.jspostcss-modules-values: postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
postcss-modules-local-by-default: postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
modules-extract-imports: postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
postcss-modules-scope: postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
postcss-import-parser: postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
postcss-icss-parser: postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
postcss-url-parser: postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
Hash: c303806b9efab4bf5dec
Version: webpack 4.44.2
Time: 4583ms
Built at: 11/17/2021 11:13:58 AM
                                       Asset      Size     Chunks                                Chunk Names
        ../../../../licenses/LICENSES-ui.txt  52.8 KiB             [emitted]
airflowDefaultTheme.9ef6a9e2f0de25c0b346.css   102 KiB          0  [emitted] [immutable]         airflowDefaultTheme
 airflowDefaultTheme.9ef6a9e2f0de25c0b346.js  4.15 KiB          0  [emitted] [immutable]         airflowDefaultTheme
            bootstrap-datetimepicker.min.css  7.54 KiB             [emitted]
             bootstrap-datetimepicker.min.js  37.1 KiB             [emitted]
                 bootstrap3-typeahead.min.js    10 KiB             [emitted]
           calendar.5260e8f126017610ad73.css  1.06 KiB          1  [emitted] [immutable]         calendar
            calendar.5260e8f126017610ad73.js  15.4 KiB          1  [emitted] [immutable]         calendar
                              codemirror.css  5.79 KiB             [emitted]
                               codemirror.js   389 KiB             [emitted]              [big]
                        coffeescript-lint.js  1.43 KiB             [emitted]
      connectionForm.be3bf4692736d58cfdb0.js  12.8 KiB          2  [emitted] [immutable]         connectionForm
                                 css-lint.js  1.28 KiB             [emitted]
                             d3-shape.min.js  29.1 KiB             [emitted]
                                   d3-tip.js  8.99 KiB             [emitted]
                                   d3.min.js   148 KiB             [emitted]
                 dag.c0b8852bb690f83bb55e.js  20.4 KiB          3  [emitted] [immutable]         dag
             dagCode.98dce599559f03115f1f.js  6.48 KiB          4  [emitted] [immutable]         dagCode
     dagDependencies.c2cdb377b2d3b7be7d1b.js  10.4 KiB          5  [emitted] [immutable]         dagDependencies
                        dagre-d3.core.min.js  27.5 KiB             [emitted]
                    dagre-d3.core.min.js.map  26.3 KiB             [emitted]
                             dagre-d3.min.js   708 KiB             [emitted]              [big]
                         dagre-d3.min.js.map   653 KiB             [emitted]              [big]
               dags.0ca53db014891875da7d.css  2.59 KiB   6, 3, 18  [emitted] [immutable]         dags
                dags.0ca53db014891875da7d.js  45.9 KiB   6, 3, 18  [emitted] [immutable]         dags
                dataTables.bootstrap.min.css  4.13 KiB             [emitted]
                 dataTables.bootstrap.min.js  1.93 KiB             [emitted]
       durationChart.ca520df04ff71dd5fab9.js  5.11 KiB          7  [emitted] [immutable]         durationChart
              flash.ab8296a74435427f9b53.css  1.36 KiB          8  [emitted] [immutable]         flash
               flash.ab8296a74435427f9b53.js  4.12 KiB          8  [emitted] [immutable]         flash
              gantt.d7989000350b53dc0855.css   1.1 KiB   9, 3, 18  [emitted] [immutable]         gantt
               gantt.d7989000350b53dc0855.js    42 KiB   9, 3, 18  [emitted] [immutable]         gantt
              graph.eaba1e30424750441353.css  2.37 KiB  10, 3, 18  [emitted] [immutable]         graph
               graph.eaba1e30424750441353.js  55.5 KiB  10, 3, 18  [emitted] [immutable]         graph
                                html-lint.js  1.94 KiB             [emitted]
                  ie.fc8f40153cdecb7eb0b3.js  16.4 KiB         11  [emitted] [immutable]         ie
                          javascript-lint.js  2.11 KiB             [emitted]
                               javascript.js  37.9 KiB             [emitted]
                    jquery.dataTables.min.js  81.6 KiB             [emitted]
                                   jshint.js  1.22 MiB             [emitted]              [big]
                                json-lint.js   1.3 KiB             [emitted]
                                    lint.css  2.55 KiB             [emitted]
                                     lint.js  8.91 KiB             [emitted]
        loadingDots.e4fbfc09969e91db1f49.css  1.21 KiB         12  [emitted] [immutable]         loadingDots
         loadingDots.e4fbfc09969e91db1f49.js  4.13 KiB         12  [emitted] [immutable]         loadingDots
               main.216f001f0b6da7966a9f.css  6.85 KiB         13  [emitted] [immutable]         main
                main.216f001f0b6da7966a9f.js  16.4 KiB         13  [emitted] [immutable]         main
                               manifest.json  3.31 KiB             [emitted]
      materialIcons.e368f72fd0a7e9a40455.css   109 KiB         14  [emitted] [immutable]         materialIcons
       materialIcons.e368f72fd0a7e9a40455.js  4.13 KiB         14  [emitted] [immutable]         materialIcons
              moment.f2be510679d38b9c54e9.js   377 KiB         15  [emitted] [immutable]  [big]  moment
                               nv.d3.min.css  8.13 KiB             [emitted]
                           nv.d3.min.css.map  3.59 KiB             [emitted]
                                nv.d3.min.js   247 KiB             [emitted]              [big]
                            nv.d3.min.js.map  1.86 MiB             [emitted]              [big]
                           oss-licenses.json  66.3 KiB             [emitted]
                         redoc.standalone.js   970 KiB             [emitted]              [big]
             redoc.standalone.js.LICENSE.txt  2.75 KiB             [emitted]
                     redoc.standalone.js.map  3.23 MiB             [emitted]              [big]
             switch.3e30e60646cdea5e4216.css  2.04 KiB         16  [emitted] [immutable]         switch
              switch.3e30e60646cdea5e4216.js  4.12 KiB         16  [emitted] [immutable]         switch
                task.8082a6cd3c389845ca0c.js  5.33 KiB         17  [emitted] [immutable]         task
       taskInstances.d758e4920a32ca069541.js  33.1 KiB      18, 3  [emitted] [immutable]         taskInstances
               tiLog.fc2c3580403a943ccddb.js  23.8 KiB         19  [emitted] [immutable]         tiLog
               tree.57c43dd706cbd3d74ef9.css  1.31 KiB      20, 3  [emitted] [immutable]         tree
                tree.57c43dd706cbd3d74ef9.js  1.48 MiB      20, 3  [emitted] [immutable]  [big]  tree
             trigger.57a3ebbaee0f22bd5022.js  5.03 KiB         21  [emitted] [immutable]         trigger
        variableEdit.45c5312f076fbe019680.js  4.97 KiB         22  [emitted] [immutable]         variableEdit
                                yaml-lint.js  1.23 KiB             [emitted]
Entrypoint airflowDefaultTheme = airflowDefaultTheme.9ef6a9e2f0de25c0b346.css airflowDefaultTheme.9ef6a9e2f0de25c0b346.js
Entrypoint connectionForm = connectionForm.be3bf4692736d58cfdb0.js
Entrypoint dag = dag.c0b8852bb690f83bb55e.js
Entrypoint dagCode = dagCode.98dce599559f03115f1f.js
Entrypoint dagDependencies = dagDependencies.c2cdb377b2d3b7be7d1b.js
Entrypoint dags = dags.0ca53db014891875da7d.css dags.0ca53db014891875da7d.js
Entrypoint flash = flash.ab8296a74435427f9b53.css flash.ab8296a74435427f9b53.js
Entrypoint gantt = gantt.d7989000350b53dc0855.css gantt.d7989000350b53dc0855.js
Entrypoint graph = graph.eaba1e30424750441353.css graph.eaba1e30424750441353.js
Entrypoint ie = ie.fc8f40153cdecb7eb0b3.js
Entrypoint loadingDots = loadingDots.e4fbfc09969e91db1f49.css loadingDots.e4fbfc09969e91db1f49.js
Entrypoint main = main.216f001f0b6da7966a9f.css main.216f001f0b6da7966a9f.js
Entrypoint materialIcons = materialIcons.e368f72fd0a7e9a40455.css materialIcons.e368f72fd0a7e9a40455.js
Entrypoint moment [big] = moment.f2be510679d38b9c54e9.js
Entrypoint switch = switch.3e30e60646cdea5e4216.css switch.3e30e60646cdea5e4216.js
Entrypoint task = task.8082a6cd3c389845ca0c.js
Entrypoint taskInstances = taskInstances.d758e4920a32ca069541.js
Entrypoint tiLog = tiLog.fc2c3580403a943ccddb.js
Entrypoint tree [big] = tree.57c43dd706cbd3d74ef9.css tree.57c43dd706cbd3d74ef9.js
Entrypoint calendar = calendar.5260e8f126017610ad73.css calendar.5260e8f126017610ad73.js
Entrypoint durationChart = durationChart.ca520df04ff71dd5fab9.js
Entrypoint trigger = trigger.57a3ebbaee0f22bd5022.js
Entrypoint variableEdit = variableEdit.45c5312f076fbe019680.js
 [8] ./static/js/dag.js 8.77 KiB {3} {6} {9} {10} {18} {20} [built]
[16] ./static/js/task_instances.js 4.54 KiB {6} {9} {10} {18} [built]
[69] ./static/css/bootstrap-theme.css 50 bytes {0} [built]
[70] ./static/js/connection_form.js 7.35 KiB {2} [built]
[71] ./static/js/dag_code.js 1.09 KiB {4} [built]
[72] ./static/js/dag_dependencies.js 6.39 KiB {5} [built]
[73] multi ./static/css/dags.css ./static/js/dags.js 40 bytes {6} [built]
[76] ./static/css/flash.css 50 bytes {8} [built]
[77] multi ./static/css/gantt.css ./static/js/gantt.js 40 bytes {9} [built]
[80] multi ./static/css/graph.css ./static/js/graph.js 40 bytes {10} [built]
[83] ./static/js/ie.js 887 bytes {11} [built]
[85] ./static/css/loading-dots.css 50 bytes {12} [built]
[86] multi ./static/css/main.css ./static/js/main.js 40 bytes {13} [built]
[88] ./static/css/material-icons.css 50 bytes {14} [built]
[93] ./static/css/switch.css 50 bytes {16} [built]
    + 425 hidden modules

WARNING in configuration
The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/configuration/mode/

WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets:
  moment.f2be510679d38b9c54e9.js (377 KiB)
  tree.57c43dd706cbd3d74ef9.js (1.48 MiB)
  nv.d3.min.js (247 KiB)
  nv.d3.min.js.map (1.86 MiB)
  dagre-d3.min.js (708 KiB)
  dagre-d3.min.js.map (653 KiB)
  redoc.standalone.js (970 KiB)
  redoc.standalone.js.map (3.23 MiB)
  codemirror.js (389 KiB)
  jshint.js (1.22 MiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  moment (377 KiB)
      moment.f2be510679d38b9c54e9.js
  tree (1.48 MiB)
      tree.57c43dd706cbd3d74ef9.css
      tree.57c43dd706cbd3d74ef9.js


WARNING in webpack performance recommendations:
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!static/css/bootstrap-theme.css:
    Entrypoint mini-css-extract-plugin = *
    [0] ./node_modules/css-loader/dist/cjs.js!./static/css/bootstrap-theme.css 130 KiB {0} [built]
        + 1 hidden module
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!static/css/calendar.css:
    Entrypoint mini-css-extract-plugin = *
    [0] ./node_modules/css-loader/dist/cjs.js!./static/css/calendar.css 1.47 KiB {0} [built]
        + 1 hidden module
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!static/css/dags.css:
    Entrypoint mini-css-extract-plugin = *
    [0] ./node_modules/css-loader/dist/cjs.js!./static/css/dags.css 3.31 KiB {0} [built]
        + 1 hidden module
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!static/css/flash.css:
    Entrypoint mini-css-extract-plugin = *
    [0] ./node_modules/css-loader/dist/cjs.js!./static/css/flash.css 2.25 KiB {0} [built]
        + 1 hidden module
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!static/css/gantt.css:
    Entrypoint mini-css-extract-plugin = *
    [0] ./node_modules/css-loader/dist/cjs.js!./static/css/gantt.css 1.58 KiB {0} [built]
        + 1 hidden module
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!static/css/graph.css:
    Entrypoint mini-css-extract-plugin = *
    [0] ./node_modules/css-loader/dist/cjs.js!./static/css/graph.css 3.16 KiB {0} [built]
        + 1 hidden module
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!static/css/loading-dots.css:
    Entrypoint mini-css-extract-plugin = *
    [0] ./node_modules/css-loader/dist/cjs.js!./static/css/loading-dots.css 1.64 KiB {0} [built]
        + 1 hidden module
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!static/css/main.css:
    Entrypoint mini-css-extract-plugin = *
    [0] ./node_modules/css-loader/dist/cjs.js!./static/css/main.css 10.7 KiB {0} [built]
    [3] ./static/sort_both.png 307 bytes {0} [built]
    [4] ./static/sort_desc.png 251 bytes {0} [built]
    [5] ./static/sort_asc.png 255 bytes {0} [built]
        + 2 hidden modules
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!static/css/material-icons.css:
    Entrypoint mini-css-extract-plugin = *
    [0] ./node_modules/css-loader/dist/cjs.js!./static/css/material-icons.css 110 KiB {0} [built]
        + 1 hidden module
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!static/css/switch.css:
    Entrypoint mini-css-extract-plugin = *
    [0] ./node_modules/css-loader/dist/cjs.js!./static/css/switch.css 2.69 KiB {0} [built]
        + 1 hidden module
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!static/css/tree.css:
    Entrypoint mini-css-extract-plugin = *
    [0] ./node_modules/css-loader/dist/cjs.js!./static/css/tree.css 1.8 KiB {0} [built]
        + 1 hidden module
✨  Done in 7.53s.
```

### How to reproduce

Checkout the `main` branch locally and run the project.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
cc: @bbovenzi - I could reproduce it locally - I believe there is not an error with Breeze but it is an error in the latest `main` w/regards to javascript files/js asset wrongly addressed (but I did not dig deeper yet - maybe you will now better) - this error is apparently reproducible in virtualenv installation as well.

**Comment by bbovenzi** [comment]
I believe that I also ran into this issue recently. If so, it was introduced in #19322. Reverting that commit locally should fix things. @jhtimmins is looking into it

**Comment by potiuk** [comment]
Thanks @bbovenzi. Confirmed reverting #19322 fixes the problem. I knew you'd know faster than trying to find myself.

**Comment by francotesei** [comment]
Hi everyone! I joined the community a few days ago, when I did the setup to the local environment I saw that issue.
So, I tried to fix it. PTAL :D

**Comment by bbovenzi** [comment]
Closed via #19667
